frontend/public/images/iconDarujme.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="25" viewBox="0 0 24 25" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
soubory bych určitě pojmenovával anglicky (když to neni třeba název loga, pak klidně česky)